pkg/types/vars_source.go

```diff
@@ -66,7 +71,8 @@ type VarsSource struct {
 	SystemEnvVars     *uo.UnstructuredObject              `json:"systemEnvVars,omitempty"`
 	Http              *VarsSourceHttp                     `json:"http,omitempty"`
 	AwsSecretsManager *VarsSourceAwsSecretsManager        `json:"awsSecretsManager,omitempty"`
-	Vault             *VarsSourceVault                    `json:"vault,omitempty"`
+	GcpSecretsManager *VarsSourceGcpSecretsManager        `json:"gcpSecretsManager,omitempty"`
+	Vault *VarsSourceVault                                `json:"vault,omitempty"`
```

Comment:
can you run `go fmt ./...` on the code to fix formatting issues? I just realised that I never added this to the CI pipeline as a verification...